ipt_connect/grappelli/dashboard/management/commands/customdashboard.py

```diff
@@ -7,22 +7,24 @@
 from django.core.management.base import BaseCommand, CommandError
```

Comment:
`ipt_connect/grappelli` is a library copied to the repo from mysterious somewhere, so we should not touch it, shouldn't we?